SEED-platform/seed
# Sticky table headers

#### What's this PR do?
A number of miscellaneous fixes I've had in my working directory:
- Adds sticky table headers to most standard tables, including:
    - dataset list, dataset detail, superuser orgs, organizations page (both tables), cycles, labels, sub organizations, org members
- Includes js lint cleanup
- Fixes the lodash vulnerability, along with all node module audit issues

#### How should this be manually tested?
- Go to the pages listed above and scroll down
- Check that the GitHub vulnerable dependency warning is gone after merging

#### What are the relevant tickets?
#1927

#### Screenshots (if appropriate)
![Sticky headers](https://thumbs.gfycat.com/AdorableConfusedElectriceel-size_restricted.gif)
